ckanext/restricted/controller.py

```diff
@@ -307,7 +307,7 @@ def _send_organization_request(self, context):
 
         return render(
             'restricted/restricted_request_organization_result.html',
-            extra_vars={'data': data_dict, 'success': success}
+            extra_vars={'data': data_dict, 'pkg_dict': {}, 'success': success}
```

Comment:
I don't understand why we need this added here as well. I have verified that request to join organization works just fine.
`restricted_request_organization_result.html` extends `base.html` template not the `package/base.html`. I might be missing something though.

Reply:
I tested it locally and it too seemed to work which was odd

In the code however...

`restricted/restricted_request_organization_result.html`
says `{% extends "restricted/restricted_request_access_base.html" %}`

`restricted/restricted_request_access_base.html`
says `{% extends "package/base.html" %}`

So for safety, I added `pkg_dict': {}` anyway
The template definitely seems to indicate needing it

Reply:
Sorry, I have commented under the wrong chaneg. I've repeat my comment in the proper place. Please have a look.